### PR TITLE
updated release note

### PR DESCRIPTION
**Pull request by JesseGoodier**
## What does this PR change?
Update release note with more information, only show EKS storage warning when PV is required and no EBS controller is running.

## Does this PR rely on any other PRs?
No

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
NA

## What risks are associated with merging this PR? What is required to fully test this PR?
Potentially bad logic in uknown EKS configs

## How was this PR tested?
Local helm installs

### CODE REVIEW

**Review by JesseGoodier — COMMENTED** [review]
Wording approved by Kai.

File: cost-analyzer/templates/NOTES.txt [hunk 1]
```diff
@@ -1,44 +1,20 @@
...
-Please allow 5-10 minutes for Kubecost to gather metrics.
-
-When configured, cost reconciliation with cloud provider billing data will have a 48 hour delay.
+Kubecost 2.0 is a major upgrade from previous versions and includes major new features including a brand new API Backend. Please review the following documentation to ensure a smooth transition: https://docs.kubecost.com/v2.0/
+For the full list of enhancements, please see our release notes: https://github.com/kubecost/cost-analyzer-helm-chart/releases/tag/v2.0.0
 
 When pods are Ready, you can enable port-forwarding with the following command:
 
-    kubectl port-forward --namespace {{ .Release.Namespace }} deployment/{{ template "cost-analyzer.fullname" . }} {{ $servicePort }}
+    kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
```
Comment (chipzoller) [inline cost-analyzer/templates/NOTES.txt]: Why was this changed to hard-code values?

Comment (jessegoodier) [reply cost-analyzer/templates/NOTES.txt]: good catch!!! copy paste error

**Review by chipzoller — APPROVED** [review]
This PR needs labels.
